Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,169 @@
+# Hires T2w MNI Template
+T2wTemplate="${HCPPIPEDIR_Templates}/MNI152_T2_0.8mm.nii.gz"
+# Hires T2w brain extracted MNI Template
+T2wTemplateBrain="${HCPPIPEDIR_Templates}/MNI152_T2_0.8mm_brain.nii.gz"
+# Lowres T2w MNI Template
+T2wTemplate2mm="${HCPPIPEDIR_Templates}/MNI152_T2_2mm.nii.gz"
+# Hires MNI brain mask template
+TemplateMask="${HCPPIPEDIR_Templates}/MNI152_T1_0.8mm_brain_mask.nii.gz"
+# Lowres MNI brain mask template
+Template2mmMask="${HCPPIPEDIR_Templates}/MNI152_T1_2mm_brain_mask_dil.nii.gz"
+# FNIRT 2mm T1w Config
+FNIRTConfig="${HCPPIPEDIR_Config}/T1_2_MNI152_2mm.cnf"
+
+##################################################################################################
+# Input variables used by PostFreesurferPipeline (longitudinal mode)
+##################################################################################################
+SurfaceAtlasDIR="${HCPPIPEDIR_Templates}/standard_mesh_atlases"
+GrayordinatesSpaceDIR="${HCPPIPEDIR_Templates}/91282_Greyordinates"
+GrayordinatesResolutions="2" #Usually 2mm, if multiple delimit with @, must already exist in templates dir
+HighResMesh="164" #Usually 164k vertices
+LowResMeshes="32" #Usually 32k vertices, if multiple delimit with @, must already exist in templates dir
+SubcorticalGrayLabels="${HCPPIPEDIR_Config}/FreeSurferSubcorticalLabelTableLut.txt"
+FreeSurferLabels="${HCPPIPEDIR_Config}/FreeSurferAllLut.txt"
+ReferenceMyelinMaps="${HCPPIPEDIR_Templates}/standard_mesh_atlases/Conte69.MyelinMap_BC.164k_fs_LR.dscalar.nii"
+RegName="MSMSulc" #MSMSulc is recommended, if binary is not available use FS (FreeSurfer)
+
+# Requirements for this script
+#  installed versions of: FSL, Connectome Workbench (wb_command)
+#  environment: HCPPIPEDIR, FSLDIR, CARET7DIR
+
+#fslsub queue
+#QUEUE=short.q
+#top level parallelization
+
+QUEUE="long.q"
+
+#pipeline level parallelization
+#parallel mode: FSLSUB, BUILTIN, NONE
+parallel_mode=BUILTIN
+
+if [ "$parallel_mode" != FSLSUB ]; then #fsl_sub does not allow nested submissions
+if [[ "${command_line_specified_run_local}" == "TRUE" || "$QUEUE" == "" ]] ; then
+    echo "About to locally run ${HCPPIPEDIR}/PostFreeSurfer/PostFreeSurferPipelineLongLauncher.sh"
+#    #NOTE: fsl_sub without -q runs locally and captures output in files
+    queuing_command=("$FSLDIR/bin/fsl_sub")
+     QUEUE=long.q
+else
+    echo "About to use fsl_sub to queue ${HCPPIPEDIR}/FreeSurfer/PostFreeSurferPipelineLongLauncher.sh"
+    queuing_command=("$FSLDIR/bin/fsl_sub" -q "$QUEUE")
+    QUEUE=""    
+fi
+else
+    queuing_command=""
+fi
+
+# This setting is for BUILTIN mode. Set to -1 to auto-detect the # of CPU cores on the node where each per-subject job is run.
+# Note that in case when multiple subject jobs are run on the same node and are submitted 
+# in parallel by e.g. fsl_sub, max_jobs should be set manually to not significantly exceed
+# (number of available cores)/(number of subjects) in the batch. 
+max_jobs=-1
+max_jobs=4
```

Comment:
Total number of subjects is not generally relevant, queue schedulers will already delay the launch of another job until there are "slots" free on the node, to avoid excessive running threads.  Rephrased in more general terms.
```suggestion
# This setting is for BUILTIN mode. Set to -1 to auto-detect the # of CPU cores on the node where each per-subject job is run.
# Note that in case when multiple subject jobs are run on the same node and are submitted 
# in parallel by e.g. fsl_sub, it may be inefficient to have the 
# (number of fsl_sub jobs running per node) * (BUILTIN parallelism) substantially exceed
# the number of cores per node
max_jobs=-1
#max_jobs=4
```

Reply:
ok